py-polars/pyproject.toml

```diff
@@ -85,7 +88,7 @@ all = [
 ]
 
 [tool.maturin]
-include = [{ path = "rust-toolchain.toml", format = "sdist" }]
+include = [{ path = "rust-toolchain.toml", format = "sdist" }, { path = "polars/**/*", format = ["sdist", "wheel"] }]
```

Comment:
Why is this necessary? Don't we update `py-polars/pyproject.toml` appropriately anyway?